core/arch/disassemble_shared.c

```diff
@@ -312,6 +312,11 @@ opnd_base_disp_disassemble(char *buf, size_t bufsz, size_t *sofar INOUT, opnd_t
                 else if (!TEST(DR_DISASM_ARM, DYNAMO_OPTION(disasm_mask)))
                     print_to_buffer(buf, bufsz, sofar, "+");
             }
+        } else if (TEST(DR_DISASM_ATT, DYNAMO_OPTION(disasm_mask))) {
```

Comment:
It seems like this should instead be dealing with the IF_NOT_X86 above and its comment and using that same code if we're going this route, rather than adding separate code down here.  It says up there it's worried about breaking existing scripts.

Reply:
I wanted to make it distinct based on syntax (intel vs. att). The intel version follows a complete different way to print the instruction (disp like [reg +/- disp], while att does disp(reg). The arm path hacks into the intel syntax path.